apps/create-app.md

```diff
@@ -18,20 +18,19 @@ After successful authorisation of your app, you will be assigned an **Access tok
 
 ## Get your credentials
 
-1. Within the administration area of your test shop, click **Apps & Themes**.
-In the drop down menu, choose **App Store**.
+1. In the sidebar of your shop’s administration area, click **Apps**.  
```

Comment:
It's still a developer test shop, isn't it ;-) Not sure, if we still wanna state that?